src/test/ui/higher-rank-trait-bounds/issue-88586-hr-self-outlives-in-trait-def.rs

```diff
@@ -0,0 +1,11 @@
+// Regression test for #88586: a higher-ranked outlives bound on Self in a trait
+// definition caused an ICE when debug_assertions were enabled.
+//
+// The error output is incidentally unhelpful; this should be improved.
```

Comment:
Adding `FIXME: ` prefix would be helpful as we can pick it up by grepping in the future.